ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -286,6 +286,7 @@ def clamp(input, min=None, max=None, *, out=None):
 clip = clamp
```

Comment:
When I run this function's test I don't get any error related to `bfloat16`. Instead, I get `KeyError: 'cpu'` in the paddle backend, which is not getting resolved by your changes.